source/help/messaging/attaching-files.rst

```diff
@@ -3,62 +3,77 @@ Attaching Files
 
-Drag and Drop
-~~~~~~~~~~~~~
-Upload a file or selection of files by dragging the files from your
-computer into the right-hand sidebar or center pane. Dragging and
-dropping attaches the files to the message input box. After dropping files, can
-optionally type a message and press **ENTER** to post.
+-  Click the attachment icon
+-  Drag and Drop
+-  Paste from the clipboard
 
 Attachment Icon
 ~~~~~~~~~~~~~~~
-Alternatively, upload files by clicking the grey paperclip icon inside
-the message input box. This opens up your system file viewer where you
-can navigate to the desired files and then click **Open** to upload the
-files to the message input box. Optionally type a message and then press
-**ENTER** to post.
+You can upload files by clicking the paperclip icon inside the message
+input box. This opens up your system file viewer where you can navigate
+to the desired files. Double-click the file or click **Open** to upload 
+the chosen files to the message input box. Then, press ENTER to post.
+
+[screenshot of paperclip icon]
 
-Pasting Images
+Drag and Drop
+~~~~~~~~~~~~~
+You can attach files by dragging them from your computer into the 
+message input box either on the right-hand sidebar or the center pane.
+After dropping the files, press ENTER to post.
+
+Paste from the Clipboard
 ~~~~~~~~~~~~~~
-On Chrome and Edge browsers, it is also possible to upload files by
-pasting them from the clipboard. This is not yet supported on other
-browsers.
+If you use Chrome or Edge browsers, you can also upload files by 
+pasting them from the clipboard. This feature is not yet supported on 
+other browsers.
 
 File Previewer
 --------------
-Mattermost has a built in file previewer that is used to view media,
-download files and share public links. Click the thumbnail of an
-attached file to open it in the file previewer.
+Mattermost has a built-in file previewer that you can use to:
 
-Sharing Public Links
-~~~~~~~~~~~~~~~~~~~~
-Public links allow you to share file attachments with people outside
-your Mattermost team. Open the file previewer by clicking on the
-thumbnail of an attachment, then click **Get Public Link**. This opens a
-dialog box with a link to copy. When the link is shared and opened by
-another user, the file will automatically download.
+-  Download files
+-  Share public links
+-  View media
 
-If **Get Public Link** is not visible in the file previewer and you
-prefer the feature enabled, you can request that your System Admin
-enable the feature from the System Console under **Security** > **Public
-Links**.
+Click the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-Download an attached file by clicking the download icon next to the file
-thumbnail or by opening the file previewer and clicking **Download**.
+You can download an attached file in two ways:
+
+-  click the download icon next to the file thumbnail
+-  open the file previewer and click **Download**
+
```

Comment:
In this case, it's probably better to just mention the first method only. It's the simplest (requires only one click) and people can discover the other method easily if they open the file previewer. Plus it's already covered by the File Previewer section of this doc.